homeassistant/components/mqtt/strings.json

```diff
@@ -116,6 +116,22 @@
       },
       "entry_type": "MQTT Device",
       "step": {
+        "availability": {
+          "title": "Availability options",
+          "description": "The availability feature allows a device to reported its availability.",
+          "data": {
+            "availability_topic": "Availability topic",
+            "availability_template": "Availability template",
+            "payload_available": "Payload available",
+            "payload_not_available": "Payload not available"
+          },
+          "data_description": {
+            "availability_topic": "Topic to receive the availabillity payload on",
+            "availability_template": "A [template](https://www.home-assistant.io/docs/configuration/templating/#using-templates-with-the-mqtt-integration) to render the availability payload received on the availability topic",
+            "payload_available": "The payload that indicates the device is available (defaults to 'online')",
+            "payload_not_available": "The payload that indicates the device is not available (defaults to 'offline')"
```

Comment:
I think we should default to online/offline in the schema too?

Reply:
That has already been added:
https://github.com/home-assistant/core/pull/138673/files#diff-ed954233b966f5069d53567f6826d3f77d0072adfbbbbfcc5ef925ad1e374d8aR235-R241